### PR TITLE
Make attestation a single step

### PR DESCRIPTION
**Pull request by CodeSandwich**
The attestation flow is reduced from 3 popups to just one

### CODE REVIEW

**Review by juliendonck — CHANGES_REQUESTED** [review]
Looks great and works as expected. The only thing I'm wondering is if we could change the CTA while the transaction is being confirmed. Cause now it says "link your ID" while it's confirming you linking your ID. Would be cool to disable the CTA or put some kind of spinner there instead of the button

Screenshot 2021-05-17 at 11 55 30

File: ui/Modal/Funding/LinkAddress.svelte [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <script lang="typescript">
-  import EthToRadicle from "./Link/EthToRadicle.svelte";
-  import SavedToRadicle from "./Link/SavedToRadicle.svelte";
-  import RadicleToEth from "./Link/RadicleToEth.svelte";
   import { Remote } from "../../DesignSystem/Component";
 
+  import { Avatar, Button, Emoji, Icon } from "../../DesignSystem/Primitive";
+  import { Copyable, TxButton } from "../../DesignSystem/Component";
```
Comment (geigerzaehler) [inline ui/Modal/Funding/LinkAddress.svelte:5]: It’s now possible (and preferable) to use absolute imports from the Design System.

```suggestion
  import { Avatar, Button, Emoji, Icon } from "ui/DesignSystem/Primitive";
  import { Copyable, TxButton } from "ui/DesignSystem/Component";
```

Comment (CodeSandwich) [reply ui/Modal/Funding/LinkAddress.svelte]: Fixed

**Comment by CodeSandwich** [comment]
Added a spinner after attestation request

**Review by geigerzaehler — DISMISSED** [review]

**Review by juliendonck — APPROVED** [review]
LGTM !

**Comment by juliendonck** [comment]
@CodeSandwich Nice work, let's merge it :)

**Comment by CodeSandwich** [comment]
Squashed and rebased, ready to merge

**Review by juliendonck — APPROVED** [review]

**Pull request merged**